layout/_partial/footer.swig

```diff
@@ -12,12 +12,13 @@
   </span>
 
   <span class="copyright-year">
-    {% set current = date(Date.now(), "YYYY") %}
-    &copy; 
-    {% if theme.since and theme.since != current %} 
-      {{ theme.since }} - 
-    {% endif %}
-    {{ current }}
+    {%- set current = date(Date.now(), "YYYY") -%}
+    &copy;
+    {%- if theme.since and theme.since != current -%}
+      {{ theme.since }} - {{ current }}
```

Comment:
Here I have to modify a bit to make generated web page text looks like `2018 - 2019` instead of `2018 -2019`.